sdl_android/src/main/java/com/smartdevicelink/streaming/VideoStreamingParams.java

```diff
@@ -0,0 +1,90 @@
+package com.smartdevicelink.streaming;
+
+import android.content.Intent;
+import android.util.DisplayMetrics;
+
+import com.smartdevicelink.proxy.rpc.ImageResolution;
+import com.smartdevicelink.proxy.rpc.VideoStreamingFormat;
+import com.smartdevicelink.proxy.rpc.enums.VideoStreamingCodec;
+import com.smartdevicelink.proxy.rpc.enums.VideoStreamingProtocol;
+
+public class VideoStreamingParams {
+	private final VideoStreamingProtocol DEFAULT_PROTOCOL = VideoStreamingProtocol.RAW;
+	private final VideoStreamingCodec DEFAULT_CODEC = VideoStreamingCodec.H264;
+	private final int DEFAULT_WIDTH = 800;
+	private final int DEFAULT_HEIGHT = 480;
+	private final int DEFAULT_DENSITY = DisplayMetrics.DENSITY_HIGH;
+	private final int DEFAULT_FRAMERATE = 24;
+	private final int DEFAULT_BITRATE = 512000;
+	private final int DEFAULT_INTERVAL = 5;
+
+
+	private int displayDensity = DEFAULT_DENSITY;
+	private int frameRate = DEFAULT_FRAMERATE;
+	private int bitrate = DEFAULT_BITRATE;
+	private int interval = DEFAULT_INTERVAL;
+	private VideoStreamingFormat format;
+	private ImageResolution resolution;
+
+    public VideoStreamingParams(){
+	    format = new VideoStreamingFormat();
+	    format.setProtocol(DEFAULT_PROTOCOL);
+	    format.setCodec(DEFAULT_CODEC);
+	    resolution = new ImageResolution();
+	    resolution.setResolutionWidth(DEFAULT_WIDTH);
+	    resolution.setResolutionHeight(DEFAULT_HEIGHT);
+    }
+
+    public VideoStreamingParams(ImageResolution resolution, VideoStreamingFormat format){
```

Comment:
This should include all parameters. Even though this works for the use case of the PR it's not a good design to create constructors for specific use cases.